ARCs/arc-0004.md

```diff
@@ -42,6 +42,8 @@ as described in [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt).
 
 > Comments like this are non-normative.
 
+All strings of type `string` are UTF-8 encoded.
```

Comment:
I was thinking the `string` in the TS interfaces. But what about the strings in the ABI?

Reply:
Got it, maybe this would be more clear?

```suggestion
Interfaces are defined in TypeScript. All the objects that are defined are valid JSON objects, and all JSON `string` types are UTF-8 encoded.
```

And ABI strings are "assumed to contain UTF-8 encoded content."